src/t2c.c

```diff
@@ -135,6 +135,9 @@ FORCE_INLINE void t2c_gen_call_io_func(LLVMValueRef start,
                    &io_param, 1, "");
 }
 
+static LLVMTypeRef t2c_cache_func_proto;
+static LLVMTypeRef jit_cache_ty;
```

Comment:
What is `ty`?

Reply:
It is common to use `ty` for the short term of `type` in Python or other languages. Should we keep the word `type` ?

Reply:
Use `type` instead.